db/seed.js

```diff
@@ -66,7 +68,7 @@ Promise.all(
 )
 .then(values => { console.log('Data generated successfully.') })
 .catch(error => { console.error(error.message) })
-connection.end( () => { console.log('Connection closed.') });
+db.end( () => { console.log('Connection closed.') });
```

Comment:
Does db.end run synchronously or asynchronously? It's my understanding that calling `.end` closes the connection after all pending queries have been resolved. Since your previous `Promise.all` block is asynchronous, how can you be certain that all the `db.query`s within `Promise.all`'s `.map` have been called before the connection closes, for all developers that would use this script in the future? 

Suggestion: move `db.end` call to inside the `.then` on line 69 AND the `.catch` on line 70 to ensure the connection closes exactly when you want it to.

Reply:
I ended up just making a second .then() after both of the others and putting it there.